Stratis.Bitcoin.Tests/NodeConfiguration/TextFileConfigurationTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿using Stratis.Bitcoin.Configuration;
+using Xunit;
+
+namespace Stratis.Bitcoin.Tests.NodeConfiguration
+{
+    public class TextFileConfigurationTests
+    {
+
+        [Fact]
+        public void GetAllWithArrayArgs()
+        {
+            // Arrange
+            var SUT = new TextFileConfiguration(new[] { "test" });
+            // Act
+            var parsed = SUT.GetAll("test");
```

Comment:
[Coding style](https://github.com/stratisproject/StratisBitcoinFullNode/blob/master/Documentation/coding-style.md).#10

multiple instances in this file